carsus/io/output/base.py

```diff
@@ -185,20 +197,28 @@ def _create_metastable_flags(levels, lines,
 
     @staticmethod
     def _create_einstein_coeff(lines):
+        """
+        Create Einstein coefficients columns for the `lines` DataFrame.
+
```

Comment:
Parameters missing. Description should include latex?

Reply:
also link to rybicki page

Reply:
Probably is better to write something in Carsus docs and put the link in the docstring.

Reply:
Let's put it here and (later) in the docs.

Reply:
added here https://github.com/tardis-sn/carsus/issues/207